src/labels/CustomLabels.labels

```diff
@@ -2127,6 +2135,38 @@
         <shortDescription>Plural Picklist substitute for Yearly</shortDescription>
         <value>Years</value>
     </labels>
+    <labels>
+        <fullName>RD2_EntryFormSaveCommitmentMessage</fullName>
+        <categories>Recurring-Donations, Entry-Form</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Spinner text when Elevate Commitment is being created</shortDescription>
+        <value>Creating Elevate commitment... This might take some time, please wait.</value>
+    </labels>
+    <labels>
+        <fullName>RD2_EntryFormSaveCommitmentFailedMessage</fullName>
+        <categories>Recurring-Donations, Entry-Form</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Message displayed as prefix to errors returned from Elevate API</shortDescription>
+        <value>However, a Commitment in Elevate could not be created due to following error(s):</value>
```

Comment:
```suggestion
        <value>However, an Elevate recurring payment could not be created due to the following errors:
</value>
```
Is it possible to append "Contact your system administrator for help." after the list of errors from Elevate? Just thinking that I want to give the user some guidance about what to do here since this means that you'd have an RD record in Salesforce, but no Elevate commitment and therefore no actual money will be collected.

Reply:
@sam-knox , does this look ok? 
<img width="1552" alt="Screen Shot 2020-09-21 at 2 35 47 PM" src="https://user-images.githubusercontent.com/28743286/93824175-d965db80-fc17-11ea-9c8e-99c8192565ba.png">

Reply:
@RadSFDeveloper - Yep, I think that'll work!

Reply:
@force2b , @sam-knox : the code is ready for review. I have made changes to the error handling on commitment creation so an error is displayed on various stages of the commitment creation and the modal closes so that user is prevented creating the same Recurring Donation in NPSP twice.

<img width="1484" alt="CommitmentUnexpectedException" src="https://user-images.githubusercontent.com/28743286/93830694-c6f29e80-fc25-11ea-9a42-70a5795fe7d8.png">

<img width="1495" alt="CommitmentAPIReturnsErrorResponse" src="https://user-images.githubusercontent.com/28743286/93830706-cd811600-fc25-11ea-8cda-0c248809a499.png">

<img width="1495" alt="CommitmentAuraEnabledError" src="https://user-images.githubusercontent.com/28743286/93830713-d2de6080-fc25-11ea-9e27-806332bf54b4.png">

<img width="1493" alt="CommitmentRequestBodyFailed" src="https://user-images.githubusercontent.com/28743286/93830739-e8538a80-fc25-11ea-85aa-e07796194104.png">

<img width="1492" alt="CommitmentInvalidRequest" src="https://user-images.githubusercontent.com/28743286/93830751-ef7a9880-fc25-11ea-8ab1-3f950fa88981.png">